nobl9/resource_role_binding.go

```diff
@@ -186,3 +206,7 @@ func resourceRoleBindingDelete(ctx context.Context, d *schema.ResourceData, meta
 	}
 	return nil
 }
+
+func valueToPointer[T any](v T) *T {
+	return &v
+}
```

Comment:
Could you explain why this func is necessary? I don't understand why using a plain ampersand isn't enough, maybe there's room for some improvement for me here :)

Reply:
Lol, my bad. That was a leftover from a different approach. I later changed the logic a bit and forgot that this is not needed anymore :) Thanks for catching that up.